AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/StorageEngine+SyncRequirement.swift

```diff
@@ -67,9 +67,9 @@ internal extension AuthRule {
         switch provider {
         // OIDC and Function providers don't need
         // Auth plugin
-        case .oidc, .function:
+        case .oidc, .function, .none:
```

Comment:
Aha! I knew it was a good idea not to have a default case :)

Reply:
:)